.github/workflows/development.yml

```diff
@@ -2,8 +2,10 @@ name: Publish Development Helm Charts
 on: workflow_dispatch
 
 jobs:
-  placeholder:
-    runs-on: ubuntu-latest
-    steps:
-      - name: test action
-        run: echo "this is a test"
\ No newline at end of file
+  call-workflow-passing-data:
+    uses: GlueOps/github-workflows/.github/workflows/publish-metacontroller-helm-chart.yml@v20230403
```

Comment:
I would point it at main so that all the actions get updated at the same time. If we need to do a hotfix you will always have the tag for the date you could revert to. By looking at the last "action" for the "tag" that was built you will be able to identify which tag to use since it's versioned by date.

Reply:
sounds good - updated